core/src/com/threecubed/auber/World.java

```diff
@@ -436,4 +443,35 @@ public void checkForEndState() {
       game.setScreen(new GameOverScreen(game, true));
     }
   }
+
+  public enum Abilities {
```

Comment:
Any particular reason this is in `World.java` and not in its own file?

Reply:
Main reason was SystemStates & Tiles enums were not in their own files

Reply:
Has been moved in latest commit